Doc/faq/windows.rst

```diff
@@ -86,59 +75,26 @@ key down while you enter a :kbd:`Z`, then hit the ":kbd:`Enter`" key to get back
 Windows command prompt.
```

Comment:
Maybe in addition to `Ctrl+Z` for exiting, it should mention typing `quit()` at the `>>>`?